transform/snowflake-dbt/models/hightouch/blapi/blapi_onprem_opportunity.sql

```diff
@@ -32,7 +32,7 @@ WITH onprem_opportunities_to_sync AS (
         ON customers_with_onprem_subs.account_external_id = account.dwh_external_id__c
     WHERE opportunity.id IS NULL
         AND NOT customers_with_onprem_subs.is_renewed 
-        AND (customers_with_onprem_subs.account_sfid is null OR customers_with_onprem_subs.account_type <> 'Customer') -- removing renewals which are part of another model
+        AND (customers_with_onprem_subs.account_sfid is null OR customers_with_onprem_subs.account_type not in ('Customer','Customer(Attrited)')) -- removing renewals which are part of another model
```

Comment:
@munish7771 - "Attrited"?

Reply:
when account type is customer or customer(Attrited), it becomes a renewal subscription.
removing them from new subscription model.